### PR TITLE
User GET - GetUserList

### LINKED ISSUE #304: Given a building, when the user GET list endpoint is called, then the list of all users should be retrieved from the database.
#272

### PR DESCRIPTION
**Pull request by roguedeleter**
Closes #304 

This PR implements the GetUserList endpoint.

### CODE REVIEW

File: BEIMA.Backend/UserFunctions/GetUserList.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using MongoDB.Bson.Serialization;
+using System.Collections.Generic;
+
+namespace BEIMA.Backend.UserFunctions
+{
+    /// <summary>
+    /// Handles a get request involving multiple buildlings.
```
Comment (josephmoore99) [inline BEIMA.Backend/UserFunctions/GetUserList.cs:13]:
```suggestion
    /// Handles a get request involving multiple users.
```

Comment (josephmoore99) [reply BEIMA.Backend/UserFunctions/GetUserList.cs:13]: Looks like I misspelled buildings on the other endpoint lol.

**Review by AshlynAdamson — APPROVED** [review]

File: BEIMA.Backend/UserFunctions/GetUserList.cs (hunk 1 above)
Comment (roguedeleter) [reply BEIMA.Backend/UserFunctions/GetUserList.cs]: Lol thanks. I went ahead and fixed this typo, as well as the original typo in GetBuildingList.cs.

**Comment by roguedeleter** [comment]
All open issues resolved

**Review by AshlynAdamson — APPROVED** [review]

**Review by josephmoore99 — APPROVED** [review]

**Pull request merged**